cmd/ido2icingadb/embed/ido_migration_progress_schema.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS ido_migration_progress (
+    history_type VARCHAR(63) NOT NULL,
+    last_ido_id  BIGINT      NOT NULL,
+
+    CONSTRAINT pk_ido_migration_progress PRIMARY KEY (history_type)
+);
```

Comment:
In the current state, given that this is written to the target database, you can only ever migrate one environment into a single Icinga DB database.

Reply:
Shall I fix it?

Reply:
Should just needs an additional `environment_id` column I guess, so yes.

Reply:
```
MariaDB [idb]> select history.event_type,count(*) from history group by history.event_type;
+----------------+----------+
| event_type     | count(*) |
+----------------+----------+
| notification   |     1423 |
| state_change   |      385 |
| downtime_start |      252 |
| downtime_end   |      137 |
| comment_add    |      144 |
| comment_remove |      144 |
| flapping_start |       88 |
| flapping_end   |       72 |
| ack_set        |      149 |
| ack_clear      |      149 |
+----------------+----------+
10 rows in set (0,004 sec)

MariaDB [idb]> select history.event_type,count(*) from history group by history.event_type;
+----------------+----------+
| event_type     | count(*) |
+----------------+----------+
| notification   |     2846 |
| state_change   |      770 |
| downtime_start |      504 |
| downtime_end   |      274 |
| comment_add    |      288 |
| comment_remove |      288 |
| flapping_start |      176 |
| flapping_end   |      144 |
| ack_set        |      298 |
| ack_clear      |      298 |
+----------------+----------+
10 rows in set (0,010 sec)

MariaDB [idb]>
```